### PR TITLE
frontmatter Java categories

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/platforms/java/config.yml [hunk 1]
```diff
@@ -2,3 +2,6 @@ title: Java
 caseStyle: camelCase
 supportLevel: production
 sdk: sentry.java
+categories:
+  - desktop
+  - server
```
Comment (marandaneto) [inline src/platforms/java/config.yml:7]: we could also add `serverless` but we don't have official docs about it yet, same for Dart, so I guess its not a good idea.

**Review by philipphofmann — APPROVED** [review]
LGTM.

File: src/platforms/java/config.yml (hunk 1 above)
Comment (philipphofmann) [reply src/platforms/java/config.yml:7]: I wouldn't add serverless then.

**Review by bruno-garcia — APPROVED** [review]
LGTM

**Pull request merged**